server/src/test/java/io/crate/types/BooleanTypeTest.java

```diff
@@ -39,10 +39,22 @@ public void test_cast_text_to_boolean() {
         assertThat(BooleanType.INSTANCE.implicitCast("FALSE"), is(false));
         assertThat(BooleanType.INSTANCE.implicitCast("f"), is(false));
         assertThat(BooleanType.INSTANCE.implicitCast("F"), is(false));
+        assertThat(BooleanType.INSTANCE.implicitCast("no"), is(false));
+        assertThat(BooleanType.INSTANCE.implicitCast("NO"), is(false));
+        assertThat(BooleanType.INSTANCE.implicitCast("n"), is(false));
```

Comment:
also `N`?

Reply:
I adjusted the test cases to be more complete